### PR TITLE
Manual integration

### PR DESCRIPTION
**Pull request by welzie**
* removes travis config
* adds a README with info about this repo and deployment instuctions
* deploy-to-wiki.sh that will push repo to upstream wiki (makes visible on wiki site)

### CODE REVIEW

File: deploy-to-wiki.sh [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+upstream="git@github.com:LaunchCodeEducation/intro-to-professional-web-dev.wiki.git"
+
+if [ $(git config --get remote.upstream.url) != "$upstream" ]; then
+    echo "Add remote named upstream by running:"
+    echo "git remote add upstream git@github.com:LaunchCodeEducation/intro-to-professional-web-dev.wiki.git"
+else
+    echo "Updating with master then deploying"
+    git checkout master
+    git fetch origin
+    git fetch upstream
```
Comment (chrisbaylewis) [inline deploy-to-wiki.sh:19]: Should this do a pull on both origin and upstream? There could be unmerged changes in both.

File: README [hunk 2]
```diff
@@ -3,7 +3,7 @@ NOTE: This repo does NOT have a .md extension, so that it does NOT show up on th
 This repo exists to create feature branches and PRs for
 the Intro to Professional Web Dev Wiki https://github.com/LaunchCodeEducation/intro-to-professional-web-dev/wiki.
 Commits on this repo need to be pushed to the repo behind https://github.com/LaunchCodeEducation/intro-to-professional-web-dev/wiki,
-which is https://github.com/LaunchCodeEducation/intro-to-professional-web-dev.wiki.git (notice the .wiki.git at the end).
+which is  (notice the .wiki.git at the end).
```
Comment (chrisbaylewis) [inline README]: This sentence needs some fixing. It lost some words in your edit.

**Review by chrisbay — APPROVED** [review]
Looks good! Pending the couple of suggested changes, this is good to merge.

File: README (hunk 2 above)
Comment (welzie) [reply README]: I removed that sentence

File: deploy-to-wiki.sh (hunk 1 above)
Comment (welzie) [reply deploy-to-wiki.sh:19]: I added merge for origin

**Pull request merged**